packages/@coorpacademy-components/src/variables/courses.ts

```diff
@@ -0,0 +1,3 @@
+const COURSE_TYPES = ['podcast', 'video', 'interactive', 'document'];
```

Comment:
Expose it from there instead: https://github.com/CoorpAcademy/components/blob/b80d364631ec2ef4a5231a4bf03d58612cb5b117/packages/%40coorpacademy-components/src/util/external-content.js#L9

(note, this is only the "external content" course types.)